Wabuluka/epic_mail
# 165024035 user login with fetch

#### What does this PR do?
User Login
#### Description of Task to be completed?
In this PR using javascript fetch, a user can now login after they have successfully signed up.
#### How should this be manually tested?
To test if the user has logged in, the application should redirect to the admin page and also if you check the storage in the browser inspector, you should be able to see your email and a token for authentication
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165024035/comments/201150883

@nakimera @kenneth051 @bekeplar
